ddtrace/internal/utils/version.py

```diff
@@ -47,3 +50,31 @@ def parse_version(version):
         parsed.release[1] if len(parsed.release) >= 2 else 0,
         parsed.release[2] if len(parsed.release) >= 3 else 0,
     )
+
+
+def _pep440_to_semver(version=None):
+    # type: (Optional[str]) -> str
+    # The library uses a PEP 440-compliant (https://peps.python.org/pep-0440/) versioning
```

Comment:
nit: this doc string is a bit stale now (just refers to the original use-case)

Reply:
I'll fix it in main branch https://github.com/DataDog/dd-trace-py/pull/4943 :pray: